### ISSUE TITLE
Error when check model to be submitted: AttributeError: type object 'Defaults' has no attribute 'stratification_coord'

### ISSUE THREAD

**Issue by YudiXie** [brain-score/vision#566]
I am trying to submit a model following this tutorial: https://www.brain-score.org/tutorial/deepdive_1

However, when I check when running the `model.py`. I got the following error:

I checked that my model could load correctly; I am unsure if this is due to some internal error in the brain score repo.

```
(brainscore) (base) [yu_xie@node092 yudixie_resnet50_imagenet1kpret_0_240222]$ python model.py
/om/user/yu_xie/miniconda3/envs/brainscore/lib/python3.7/site-packages/brainscore_core/metrics/__init__.py:16: FutureWarning: xarray subclass Score should explicitly define __slots__
  class Score(DataAssembly):
Loaded model from model.pth
Loaded model from model.pth
Traceback (most recent call last):
  File "model.py", line 61, in <module>
    check_models.check_base_models(__name__)
  File "/weka/scratch/weka/dicarlo/yu_xie/projects/brain-score/brainscore_vision/model_helpers/check_submission/check_models.py", line 39, in check_base_models
    check_processing(model, module)
  File "/weka/scratch/weka/dicarlo/yu_xie/projects/brain-score/brainscore_vision/model_helpers/check_submission/check_models.py", line 47, in check_processing
    benchmark = _MockBenchmark()
  File "/weka/scratch/weka/dicarlo/yu_xie/projects/brain-score/brainscore_vision/model_helpers/check_submission/check_models.py", line 73, in __init__
    if hasattr(self.assembly, brainscore_vision.metric_helpers.Defaults.stratification_coord) else None))
AttributeError: type object 'Defaults' has no attribute 'stratification_coord'
```

**Comment by mike-ferguson** [comment]
Hey @YudiXie, I will check this out too! Stay tuned

**Comment by mike-ferguson** [comment]
@YudiXie This should be addressed in #585, but I am testing that fix now. I split this issue into a new one, as the fix is on vision instead of web
